components/private_channel/browser/request_utils.cc

```diff
@@ -0,0 +1,36 @@
+  /* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <string>
+
+#include "brave/components/private_channel/browser/static_values.h"
+#include "brave/components/private_channel/browser/request_utils.h"
```

Comment:
the header for this file should be at the top

Reply:
- [x] move header for `request_utils` to top